File: Python Exams/Basic Exam Collection/Basics Online Exam - 6 and 7 July 2019/05. FootballTournament/05. FootballTournament.py
team = input()
games_played = int(input())

result_list = [input() for _ in range(games_played)]
win = result_list.count('W')
equal = result_list.count('D')
lose = result_list.count('L')

total_points = win * 3 + equal * 1 + lose * 0

if games_played >= 1:
    print(f"{team} has won {total_points} points during this season.\nTotal stats:")
    print(f"## W: {win}\n## D: {equal}\n## L: {lose}")
    print(f"Win rate: {(win / games_played) * 100 :.2f}%")
else:
    print(f"{team} hasn't played any games during this season.")
    


'''
Liverpool
10
W
D
D
W
L
W
D
D
W
W
'''    

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 July 2019/05. FootballTournament/05. FootballTournament.js
function footballTournament(arg) {
  let fClub =  arg[0]
  let match =  +arg[1]
  let [w, d, l] = [0, 0, 0]
  let [wP, dP, lP] = [0, 0, 0]
 
  if (match === 0) {
     console.log(`${fClub} hasn't played any games during this season.`)
     return
  }

for (let i = 2; i < arg.length; i++) {
  switch (arg[i]) {
  case 'W': wP += 3; w++ ; break;
  case 'D': dP += 1; d++ ; break;
  case 'L': lP += 0; l++ ; break;
  }
}

let allPoints = wP + dP + lP
let percent = (w / match ) * 100
console.log(`${fClub} has won ${allPoints} points during this season.`);
console.log(`Total stats:\n## W: ${w}\n## D: ${d}\n## L: ${l}`);
console.log(`Win rate: ${percent.toFixed(2)}%`);
}
//footballTournament(['Barcelona', '7', 'W', 'D', 'L', 'L', 'W', 'W', 'D',]) 

